python/ray/serve/task_runner.py

```diff
@@ -111,17 +114,33 @@ def _ray_serve_get_runner_method(self, request_item):
                                     "The avaiable methods are {}".format(
                                         method_name, dir(self)))
 
-        return getattr(self, method_name)
+        if method_name != "__call__":
+            return getattr(self, method_name)
```

Comment:
What's happening if `method_name` doesn't exist?

Reply:
`getattr` will error and user will see an error message